src/ui/windows/TogglDesktop/TogglDesktop/ui/ViewModels/TimelineViewModel.cs

```diff
@@ -366,14 +366,14 @@ public static void ChangeFirstEntryStop(TimeEntryBlock item, List<TimeEntryBlock
         {
             var (first, last) = GetOverlappingPair(item, blocks);
             if (last == null) return;
-            Toggl.SetTimeEntryEndTimeStamp(first.TimeEntryId, (long)last.Started - 1);

```

Comment:
@atsvirchkova this commit is "dont allow posibility of negative duration".
when can the negative duration be achieved? since Started is a UTC timestamp, it will be zero only on Jan 1st 1970, so I think we're safe here. Plus we're using -1 and +1 in the macOS app: https://github.com/toggl-open-source/toggldesktop/blob/505af8744c6dcd9736e13b7c4f9ca39529d64166/src/ui/osx/TogglDesktop/Features/Timeline/Models/TimelineData.swift#L203-L228

Reply:
sorry, I misread "negative duration" for "negative timestamp".
perhaps we can just not show these items in case the duration is zero 🤔 ?